phoible/templates/about.mako

```diff
@@ -0,0 +1,13 @@
+<%inherit file="home_comp.mako"/>
+
+<h2>About</h2>
+
+<p>
+    More information about the data curation process for the PHOIBLE database is available at
+    ${h.external_link('http://phoible.github.io/')}, in particular
+</p>
+<ul>
+    <li>An FAQ: ${h.external_link('http://phoible.github.io/faq/')}</li>
```

Comment:
@xrotwang -- this made me laugh because I even had to ask the internet if it's "a faq" or "an faq".

"It all depends on whether you pronounce FAQ as a word (fack) or the individual letters, (eff-aye-kew)."

:)

Reply:
And now I know in my dialect it's pronounced "fack".

Reply:
But let's leave it as is.